tic_tac_toe/game.py

```diff
@@ -63,22 +80,55 @@ def _check_winning_combinations(board, player):
     Returns the player (winner) of any of the winning combinations is completed
     by given player, or None otherwise.
     """
-    pass
-
+    win = False
+    for i in range(3): #Rows -- i know this is not pythonic -- thinking about how to improve    
+        if _is_winning_combination(board, ((0,i),(1,i),(2,i)), player):
+            return player
+    for i in range(3): #Columnbs -- i know this is not pythonic -- thinking about how to improve    
+        if _is_winning_combination(board, ((i,0),(i,1),(i,2)), player):
+            return player
+    if _is_winning_combination(board, ((0,0),(1,1),(2,2)), player) or _is_winning_combination(board, ((0,2),(1,1),(2,0)), player):
+        return player
+        
+    return None        
 
 # public interface
+
+
 def start_new_game(player1, player2):
     """
     Creates and returns a new game configuration.
     """
-    pass
+    gamedict = {
+    'player1': "X",
+    'player2': "O",
+    'board': [
+        ["-", "-", "-"],
+        ["-", "-", "-"],
+        ["-", "-", "-"],
+    ],
+    'next_turn': "X",
+    'winner': None
+    }
+    return gamedict
 
 
 def get_winner(game):
     """
     Returns the winner player if any, or None otherwise.
     """
-    pass
+    
+    #heck_winning_combinations(board, player):
+    if _check_winning_combinations(game['board'],game['player1']) == game['player1']:
+        game['winner'] = game['player1']
```

Comment:
Interesting that assign the winner of the game as part of the manual check function. You could potentially reduce the work done in this function by checking and assigning during the actual move